src/components/accessible-autocomplete/accessible-autocomplete.js

```diff
@@ -17,6 +17,7 @@ AccessibleAutoComplete.prototype.init = function init() {
       autoselect,
       defaultValue,
       minLength,
+      confirmOnBlur: true,
```

Comment:
just added to check, it's the default but I think I've misunderstood something about how this works - because `should select a matching option in the underlying select when input is changed if there is one` isn't working as I would expect